<launch>
    
  <!-- PrioVR main node -->
  <node name="get_joint_states" pkg="priovr_interface" type="get_joint_states.py" output="screen">
    <rosparam command="load" file="$(find priovr_interface)/config/priovr_common.yaml"/>
    <rosparam command="load" file="$(find priovr_interface)/config/human.yaml"/>
    <param name="debug" value="true"/>
  </node>

</launch>
